src/frontend/packages/lib_markdown/src/components/MarkdownEditor/index.tsx

```diff
@@ -291,25 +286,37 @@ export const MarkdownEditor = ({ markdownDocumentId }: MarkdownEditorProps) => {
             >
               {intl.formatMessage(messages.publish)}
             </Button>
-            <Box flex="shrink" margin="xxsmall" />
+
             <Button
               onClick={() => saveDocument()}
               disabled={!canSaveDocument() || !contentChanged.current}
             >
               {intl.formatMessage(messages.save)}
             </Button>
           </Box>
+
+          <Box direction="row">
+            <ScreenDispositionSelector
+              screenDisposition={screenDisposition}
+              setScreenDisposition={setScreenDisposition}
+            />
+            <Box flex="grow" />
+            <LanguageSelector
```

Comment:
It is maybe a good moment to update `LanguageSelector` to fit with the select design.
Here an example of implementation of the select with the design from the sketch:
https://github.com/openfun/marsha/blob/master/src/frontend/apps/standalone_site/src/features/Authentication/components/RenaterAuthenticator.tsx#L124

Basically it is just to integrate with the component Select, I am not sure if FormField is necessary. 
<FormField>
  <Select
   ...

Reply:
Cool, thanks :+1: